Configuration/FlexForms/ServiceBw2FlexFormConfiguration.xml

```diff
@@ -0,0 +1,150 @@
+<T3DataStructure>
+    <meta>
+        <langDisable>1</langDisable>
+    </meta>
+    <sheets>
+        <sDEFAULT>
+            <ROOT>
+                <TCEforms>
+                    <sheetTitle>LLL:EXT:service_bw2/Resources/Private/Language/FlexForms.xlf:sheet_general</sheetTitle>
+                </TCEforms>
+                <type>array</type>
+                <el>
+                    <settings.organisationseinheiten.listItems>
+                        <TCEforms>
+                            <label>
+                                LLL:EXT:service_bw2/Resources/Private/Language/FlexForms.xlf:settings.organisationseinheiten.list_items
+                            </label>
+                            <displayCond>FIELD:parentRec.CType:=:servicebw2_organizationalunitslist</displayCond>
+                            <config>
+                                <type>select</type>
+                                <enableMultiSelectFilterTextfield>true</enableMultiSelectFilterTextfield>
+                                <renderType>selectMultipleSideBySide</renderType>
+                                <itemsProcFunc>JWeiland\ServiceBw2\Tca\OrganisationseinheitenItems->getItems
+                                </itemsProcFunc>
```

Comment:
Please prevent ENTER wherever possible in FlexForm. Results in side-effects while parsing the FlexForm